appengine/standard_python3/django/polls/test_polls.py

```diff
@@ -1,10 +1,10 @@
-# Copyright 2020 Google LLC. All rights reserved.
+# Copyright 2021 Google LLC
```

Comment:
This should stay as 2020 probably.

Reply:
If I'm otherwise editing a file the licence bot will yell if the year isn't the current year. 

In theory this file doesn't have significant changes and I can back the changes in this file out entirely.